javascriptlib/src/mill/javascriptlib/TestModule.scala

```diff
@@ -25,6 +25,82 @@ trait TestModule extends TaskModule {
 object TestModule {
   type TestResult = Unit
 
+  trait Coverage extends TypeScriptModule with TestModule {
+    override def npmDevDeps: T[Seq[String]] = Task {
+      super.npmDevDeps() ++ Seq("serve@14.2.4")
+    }
+
+    private[TestModule] def runCoverage: T[TestResult]
+
+    protected def coverageTask(args: Task[Seq[String]]): Task[TestResult] = Task { runCoverage() }
+
+    def coverage(args: String*): Command[TestResult] =
+      Task.Command {
+        coverageTask(Task.Anon { args })()
+      }
+
+    // <rootDir> = '/out'; allow coverage resolve distributed source files.
+    // & define coverage files relative to <rootDir>.
+    private[TestModule] def link: Task[Unit] = Task.Anon {
```

Comment:
This name is far too short given how niche the task is. Maybe call it `coverageSetupSymlinks` or something?